README-FIPS.md

```diff
@@ -28,8 +28,12 @@ resp. `fips.dll` (on Windows). The FIPS provider does not get built and
 installed automatically. To enable it, you need to configure OpenSSL using
 the `enable-fips` option.
 
-Installing the FIPS module
-==========================
+Installing the FIPS provider
+============================
+
+You can build the OpenSSL source code and the FIPS provider by doing the following
+if you only use the FIPS validated source code. If you want to use a validated FIPS
+module with the latest release see the next section.
```

Comment:
This is a little bit confusing I think. You can always build fips provider like this but it of course won't be FIPS validated.  Perhaps:
```suggestion
You can build the OpenSSL source code with the validated FIPS provider by doing the
following if you only use the FIPS validated source code. If you want to use
a validated FIPS module with the latest release see the next section.
```